application-operator/controllers/clusters/verrazzanoproject/verrazzanoproject_controller.go

```
@@ -55,8 +58,18 @@ func (r *Reconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 		return result, client.IgnoreNotFound(err)
 	}
 
-	err = r.createOrUpdateNamespaces(ctx, vp, logger)
-	return result, err
+	opResult := controllerutil.OperationResultNone
+	if vp.Namespace == constants.VerrazzanoMultiClusterNamespace {
+		err = r.createOrUpdateNamespaces(ctx, vp, logger)
+		if err == nil {
```

Looks like we ignore any errors? Is that correct?

Perhaps this should be a check for err and return it and then the opResult below that block?

---

we don't ignore errors - we pass the err to the updateStatus function below, and that results in a status update to failure

---

Never mind - looks like the err does get set and leveraged below in the status update.  Though an "err == nil" seem un-go-like ;)

---

The opResult does not get used in the status update if err is non-nil

---

Looks like our comments crossed on the interwebs... :)